microsoft/botframework-solutions
# [Virtual Assistant Sample][TypeScript] Add new Virtual Assistant Sample

## Description
- Add `Virtual Assistant Sample`

Related #1058 

## Testing Steps

1. Go to `AI\templates\Virtual-Assistant-Template\src\typescript\sample-assistant`
2. Open a terminal in this location
3. Run the `deploy.ps1` (which is inside `deployment\scripts` folder) being in generated assistant folder
4. Verify that the deployment finished successfully
5. Run the command `npm install` and `npm run build`
6. Run the `sample-assistant` with the command `npm run start` 
7. Check that the sample has the same behavior as C# version

## Checklist
\-
